src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -120,13 +133,17 @@ public RobotMap() {
 		Component.rightWheel = new Motor("RightWheel", false, new AccelerationCap(Component.pdp),
 			new VictorSP(Port.PWM.rightDriveA), new VictorSP(Port.PWM.rightDriveB));
 		Component.rightWheel.setInverted(true);
-		// BallIO
-		Motor ballioDirectionalRoller = new Motor(new CANTalon(Port.CANMotor.ballioDirectionalRoller));
-		ballioDirectionalRoller.setInverted(true);
-		Motor ballioHopperRollers = new Motor(new CANTalon(Port.CANMotor.ballioHopperRollers));
-		ballioHopperRollers.setInverted(true);
-		Motor ballioElevatorAndIntakeRoller = new Motor(new CANTalon(Port.CANMotor.ballioElevatorAndIntakeRoller));
-		ServoSubsystem ballioDoorServo = new ServoSubsystem(new Servo(Port.PWM.ballioDoorServo));
+		// Ball-Intake-Outtake
+		Motor ballioDirectionalRoller = new Motor(new VictorSP(Port.PWM.ballioTopMotor));
```

Comment:
Suggestion: Rename the motors to match the name of the rollers they are using. For example, name the "ballioTopMotor" "ballioDirectionalMotor"